app/models/link.rb

```diff
@@ -0,0 +1,19 @@
+class Link
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  embedded_in :link_check_report
+
+  field :uri, type: String
+  field :status, type: String
+  field :checked, type: DateTime
+  field :check_warnings, type: String
```

Comment:
Should these not be arrays of strings?

Reply:
I would just stick with `type: Array` to be honest, no need to specify the element type exactly.